sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -249,18 +277,26 @@ class QueryExecution(
      * Dumps debug information about query execution into the specified file.
      *
      * @param maxFields maximum number of fields converted to string representation.
+     * @param explainMode the explain mode to be used to generate the string
+     *                    representation of the plan.
      */
-    def toFile(path: String, maxFields: Int = Int.MaxValue): Unit = {
+    def toFile(path: String,
```

Comment:
put `path: String` to the next line. See https://github.com/databricks/scala-style-guide#spacing-and-indentation